src/components/impact_picture_section/ImpactPictureSection.tsx

```diff
@@ -26,14 +26,16 @@ class ImpactPicture extends React.Component {
   render() {
     const pictureName = this.props.pictureName;
     const title = this.findRightText(this.props.text, 'title');
-    const text = this.findRightText(this.props.text, 'subtext');
+    const text1 = this.findRightText(this.props.text, 'subtext-first');
```

Comment:
This text has to be parsed to add the bold text in the first sentence like the trello card says:

https://trello.com/c/tIaefAQ2/79-make-2-paragrahps-instead-of-1-big-separate-before-organisations

Reply:
Added the bold text.  I used const boldText1 = text1.split(/\s+/).slice(0, 5).join(' '); to split it up.  Let me know if you would like it done another way.  :)

Reply:
Looks good to splitting it up on the first 5 words :)
Then it will keep doing that :)